src/electron/platform/android/setup/android-setup-steps-configs.ts

```diff
@@ -45,5 +45,6 @@ export const allAndroidSetupStepConfigs: AndroidSetupStepConfigs = {
     'prompt-install-failed': promptInstallService,
     'detect-permissions': detectPermissions,
     'prompt-grant-permissions': promptGrantPermissions,
+    'configuring-port-forwarding': null, // to be implemented in future feature work
```

Comment:
is this meant to be null?

Reply:
Yes, this is the config for the backend step; Dave will be implementing it in a separate PR

Reply:
ah, I mistook it for the UI side - thanks